lib/sorbet-rails/model_rbi_formatter.rb

```diff
@@ -84,6 +84,12 @@ def populate_activerecord_querying_methods
     # All is a named scope that most method from ActiveRecord::Querying delegate to
     # rails/activerecord/lib/active_record/querying.rb:21
     @generated_scope_sigs["all"] = { ret: "#{@model_class.name}::Relation" }
+    @generated_scope_sigs["unscoped"] = {
+      ret: "T.untyped",
```

Comment:
Oops, my mistake. I missed the comment about the block. I'm looking into the optional block and [`.scoping`](https://apidock.com/rails/ActiveRecord/Relation/scoping). Sorbet doesn't support multiple signatures for a method, but making the method return `untyped` isn't useful. I think it's more useful for this method to return a relation, and if people want to use the optional block option, they should use `.scoping`.

Eg:
```
Model.unscoped.scoping do ... end
```
We could add this guidance to the "Tips and Tricks" section.

Reply:
This method should also be added to `@generated_class_sigs` because it only exists in `ActiveRecord::Base` class and not in a `ActiveRecord::Relation`

Reply:
Edit: Never mind, I was wrong, again